### PR TITLE
4.0.0-alpha.1

### PR DESCRIPTION
**Pull request by paulirish**
might as well.

### CODE REVIEW

File: changelog.md [hunk 1]
```diff
@@ -1,3 +1,87 @@
...
+* handle underscored not_applicable scoreDisplayMode ([#6549](https://github.com/googlechrome/lighthouse/pull/6549))
+* css: make performance category more compact ([#6437](https://github.com/googlechrome/lighthouse/pull/6437))
+* rename clump classes; give classes to all audit groups ([#6482](https://github.com/googlechrome/lighthouse/pull/6482))
+* refactor rendering of top-level failed/passing/etc clumps ([#6460](https://github.com/googlechrome/lighthouse/pull/6460))
+
+## Misc
+
+* build: fix viewer deploy ([#6449](https://github.com/googlechrome/lighthouse/pull/6449))
+* timing: move timing-trace to async events ([#6440](https://github.com/googlechrome/lighthouse/pull/6440))
+* increase generic protocol timeout to 10s (#6475) ([#6505](https://github.com/googlechrome/lighthouse/pull/6505))
```
Comment (brendankenny) [inline changelog.md]: delete the `(#6475)` (my fault it's there)

Comment (brendankenny) [reply changelog.md]: or combine it with the #6531 entry above, since this was the erroneous one

File: changelog.md [hunk 2]
```diff
@@ -1,3 +1,87 @@
...
+* update puppeteer to version 1.10.0 ([#6493](https://github.com/googlechrome/lighthouse/pull/6493))
+* readme: use official appveyor badge ([#6443](https://github.com/googlechrome/lighthouse/pull/6443))
+* i18n: convert accessibility audits ([#6229](https://github.com/googlechrome/lighthouse/pull/6229))
+
+ <a name="4.0.0-alpha.1"></a>
```
Comment (brendankenny) [inline changelog.md]: changelog is doubled up

**Comment by paulirish** [comment]
Shipped. npm, cws, viewer. 4.0.0-alpha.1

**Pull request merged**